Files/Helpers/FolderGlyphAssetHelper.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Microsoft.Graphics.Canvas;
+using Microsoft.Graphics.Canvas.Svg;
+using Microsoft.Graphics.Canvas.Text;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.Foundation;
+using Windows.Storage;
+using Windows.UI;
+
+namespace Files.Helpers
+{
+    public static class FolderGlyphAssetHelper
+    {
+        public static async Task<(Uri Path150x150, Uri Path71x71)> GenerateAssetsAsync(string glyph = "")
+        {
+            var small = await GenerateAssetAsync(250, 250, 56, 0.5f, glyph);
+            var medium = await GenerateAssetAsync(300, 300, 42, 0.4f, glyph);
+            return (medium, small);
+        }
+
+        public static async Task<Uri> GenerateAssetAsync(float widthDpi, float heightDpi, float fontSize, float scale, string glyph = "")
```

Comment:
Since the glyph has been removed, do we need all this or can we simply use the existing .svg icon?

Reply:
@gave92 SVGs dont work as assets for start tiles, and this does work to make a valid png to use. However, the generated pngs could just be added as app assets instead.

If there are any plans to allow for custom icons for the tiles in the future, I think it should be kept as it can be easily adapted. Otherwise, I suppose it could be removed.

Reply:
@yaichenbaum what do you think? If we know it won't be needed for a while we can ship adding the Win.2D library as dependency

Reply:
@gave92 We are not going to need it for a while so I think it can be removed.

Reply:
Ok, I removed the asset generator along with the win2d dependency.